src/convert.js

```diff
@@ -0,0 +1,27 @@
+import JSBI from 'jsbi'
+
+export function getRawRate(convertRate) {
+  if (typeof convertRate === 'number' || typeof convertRate === 'string') {
+    return convertRate.toString().split('.')
+  }
+  return [convertRate.value, '']
```

Comment:
Rather than this, would it be better to do an `instanceof TokenAmount`? And in the case that it's not, ensure it's a string by `String(convertRate)`?

Reply:
Resolving this; due to new API changes this function was removed.